### ISSUE TITLE
Does SenCID support mouse data?

### ISSUE THREAD

**Issue by Yujj1123** [JackieHanLab/SenCID#8]
Thank you for developing this algorithm! 
I've noticed that the original article seems to include only human samples, and the tutorial doesn't mention anything about species. Could you please let me know if SenCID can be applied to mouse data?

**Comment by SherryMiyano** [comment]
You are right, currently the models are all trained by human datasets. You may convert the gene symbols to the homologs from mouse to human and run SenCID, but the accuracies of the results are not yet validated.
